tests/fixtures/utils.py

```diff
@@ -1,20 +0,0 @@
-# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
```

Comment:
For reasons beyond me, we usually keep the utils files. But also, I don't really care.